src/test/java/io/xpring/xrpl/XpringClientIntegrationTests.java

```diff
@@ -31,7 +31,7 @@
     private static final BigInteger AMOUNT = new BigInteger("1");
 
     /** Hash of a successful transaction. */
-    private static final String TRANSACTION_HASH = "24E31668208A3165E6C702CDA66425808EAD670EABCBFA6C4403FFA93500D486";
+    private static final String TRANSACTION_HASH = "6D9B6CDA7F6752548800C4FC14A037B8DAC2EF47E61028793768766EAB7FC81B";
```

Comment:
We still need to update these once a month, correct?

Reply:
Correct. https://app.asana.com/0/1143722888069470/1166110587347849